packages/google-workspace-adapter/src/adapter_creator.ts

```diff
@@ -77,6 +78,11 @@ export const adapter = createAdapter<Credentials, Options, UserConfig>({
   operationsCustomizations: {
     connectionCreatorFromConfig: () => createConnectionForApp(DIRECTORY_APP_NAME),
     credentialsFromConfig: defaultCredentialsFromConfig,
+    customizeFilterCreators: args => ({
+      ...filters.createCommonFilters<Options, UserConfig>(args),
```

Comment:
maybe we can change some logic in here: https://github.com/salto-io/salto/blob/586e3b878823e6ff6c9a22f8c6ba71edce2d01cb/packages/adapter-components/src/adapter-wrapper/adapter_creator.ts#L119 so all the adapters wont need to pass the common filter, as all of the adapter will probably have some filters.
```suggestion
    operationsCustomizations: {
        runCommonFilters: true
        customizeFilterCreators: args => ({
              // customPathsFilterCreator must run after fieldReferencesFilter
              customPathsFilterCreator,
```

Reply:
the order is important so we'd need to add before and after filters and it complicates the interface a bit. we can do something similar to what we're doing in the change validators (Farkash had a suggestion that would allow full control of the order with some games), but I thought for now it's clearer to just require the extra line